### PR TITLE
fix: add cmd line mysql ssl options, fixes RHOAIENG-3337

### PR DESCRIPTION
**Pull request by dhirajsb**

### CODE REVIEW

File: ml_metadata/metadata_store/metadata_store_server_main.cc [hunk 1]
```diff
@@ -328,6 +355,23 @@ DEFINE_string(mysql_config_user, "",
...
+              "certificate authority (CA) certificate(s).");
+DEFINE_string(mysql_config_sslcapath, "",
+              "This parameter specifies path name of the directory "
+              "that contains trusted SSL CA certificates.");
+DEFINE_string(mysql_config_sslcipher, "",
+              "This parameter specifies the list of permissible ciphers for "
+              "SSL encryption.");
+DEFINE_bool(mysql_config_verify_server_cert, false,
+              "This parameter enables verification of the server certificate "
+              " against the host name used when connecting to the server.");
```
Comment (tarilabs) [inline ml_metadata/metadata_store/metadata_store_server_main.cc:374]:
```suggestion
DEFINE_bool(mysql_config_verify_server_cert, false,
              "When certificates are provided, this parameter enables verification of the server certificate "
              " against the host name used when connecting to the server.");
```

wdyt, given this parameter takes effects only iff ssl/certificates parameters are provided?

**Review by tarilabs — APPROVED** [review]
lgtm but I have no cpp experience so I believe meaningful if @isinyaaa could chime in please :)

File: ml_metadata/metadata_store/metadata_store_server_main.cc (hunk 1 above)
Comment (dhirajsb) [reply ml_metadata/metadata_store/metadata_store_server_main.cc:374]: I think it's fine as it is. The description comes from the proto declaration anyway.

**Review by lampajr — CHANGES_REQUESTED** [review]
The change itself looks good to me but shouldn't we put this change in [opendatahub-io/ml-metadata](https://github.com/opendatahub-io/ml-metadata) first and then sync the `master` branch? As we are already doing with https://github.com/opendatahub-io/ml-metadata/pull/2

**Comment by dhirajsb** [comment]
Closing this PR to re-open it in ODH repo.

**Comment by dhirajsb** [comment]
Reopened as https://github.com/opendatahub-io/ml-metadata/pull/4

**Pull request closed without merging**